lib/urbanopt/scenario/default_reports/schema/scenario_schema.json

```diff
@@ -141,9 +143,11 @@
 			"type": "object",
 			"properties": {
 				"id": {
+					"description": "Id refers to the id of the reporting period",
```

Comment:
Need a note that ids of reporting periods are matched across simulations, so the same period of time gets the same id for all simulations